confgenerator/testdata/invalid/linux/logging-processor_exclude_logs_invalid_parse/input.yaml

```diff
@@ -0,0 +1,9 @@
+logging:
+  processors:
+    p1:
+      type: exclude_logs
+      condition: jsonPayload.message!~foo
```

Comment:
@quentinmit This one _ought to_ work but there's a bug in the grammar that I haven't gotten to the bottom of yet. Not a high priority IMO since it works once you insert spaces around the `!~` but I wonder if there are other similar failing cases.

I'm starting to remember another failing case relating to inserting spaces before closing parentheses, such as `(jsonPayload.a = b )`; I'll add a test for that too if I can reproduce it.

Reply:
Added a test for the latter.